Tools/CISupport/ews-build/steps.py

```diff
@@ -5041,8 +5041,8 @@ def run(self):
         self.addLogObserver('json', self.log_observer_json)
 
         platform = self.getProperty('platform')
-        if platform == 'gtk':
-            self.command = ['python3', 'Tools/Scripts/run-gtk-tests',
+        if platform in ['gtk', 'wpe']:
+            self.command = ['python3', 'Tools/Scripts/run-{0}-tests'.format(platform),
```

Comment:
while you are at it, can also use f-string here.

Reply:
sure! thanks for the tip 😀